couchdb-setup.sh

```diff
@@ -87,12 +87,13 @@ curl -X PUT $COUCHURL/_replicator
 curl -X PUT $COUCHURL/_global_changes
 
 # Add planet app databases
-curl -X PUT $COUCHURL/meetups
-curl -X PUT $COUCHURL/resources
-curl -X PUT $COUCHURL/courses
-curl -X PUT $COUCHURL/nations
 curl -X PUT $COUCHURL/communityregistrationrequests
+curl -X PUT $COUCHURL/courses
```

Comment:
@paulbert 
This is not a  added line. We just sorted the database alphabetically in script file.

Reply:
@paulbert is there anything required to changes?

Reply:
@badri3051 This is confusing to look at from a change log perspective.  If we were to look at this commit 3 months from now, it would take time to figure out that only one DB was added to this file.  If the only change that shows up is one line added for the ratings database we can tell what happened at a glance.